### ISSUE TITLE
You working on that? 

### ISSUE THREAD

**Issue by rhcarlosweb** [DylanVann/wordpress-popular-posts-query#1]
I really need that function haha, you working on that?

**Comment by DylanVann** [comment]
I'm sorry you had to be confronted with a description of a solution but no code haha.

I coded most of it for a project but it isn't in the form of a plugin at the moment. If you give me a day or two the fact that there's someone interested might motivate me to finally get around to packaging it up, at least a rough version of it.

**Comment by rhcarlosweb** [comment]
haha, i really really want that

**Comment by rhcarlosweb** [comment]
Because that plugin its so hard use default wp query of WordPress, and i need this forbmy projecta haha

**Comment by DylanVann** [comment]
Yeah, the plugins for popular posts make it really hard to do exactly what you want. I'll reopen this until I get some working code in this repo.

**Comment by rhcarlosweb** [comment]
Thanks i'll follow this repo :)
